backend/tournesol/admin.py

```diff
@@ -98,7 +98,7 @@ def get_publication_date(obj):
     @staticmethod
     @admin.display(description="language", ordering="metadata__language")
     def get_language(obj):
-        return obj.metadata.get("language")
+        return LANGUAGE_CODE_TO_NAME_MATCHING.get(obj.metadata.get("language"))
```

Comment:
As a precaution, the language code can still be used as a fallback.  
It would avoid confusion in case when an unknown value is present in the database, even though it should not happen given the current implementation.
```suggestion
        language_code = obj.metadata.get("language")
        return LANGUAGE_CODE_TO_NAME_MATCHING.get(language_code, language_code)
```